devvit.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://developers.reddit.com/schema/config-file.v1.json",
+  "name": "YOUR_APP_NAME",
+  "post": {
+    "client": {
+      "dir": "src/client/dist",
+      "entry": "src/client/dist/index.html"
+    },
+    "create": {
+      "height": "tall",
+      "menu": {
+        "enable": true,
+        "label": "Hello World",
+        "scope": "moderator"
```

Comment:
this _should_ be the default. suggest omitting the menu prop entirely and just embracing the defaults.